src/controllers/sign.out.controller.ts

```diff
@@ -14,7 +13,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     return res.render(config.SIGN_OUT_PAGE, {
       previousPage: previousPageUrl,
-      saveAndResume: isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022),
+      saveAndResume: true,
```

Comment:
the SIGN_OUT_PAGE is also used by the update sign out controller so the template will still need the logic until the update S&R flag is removed, then this saveAndResume: true can also be removed.

config.UPDATE_SIGN_OUT_PAGE = "sign-out", the same as config.SIGN_OUT_PAGE, no idea why another config value was needed, but they both point to the same template.